bin/brew

```diff
@@ -135,6 +135,24 @@ then
 fi
 
 HOMEBREW_COMMAND="$1"
+shift
+case "$HOMEBREW_COMMAND" in
+  ls)          HOMEBREW_COMMAND="list";;
+  homepage)    HOMEBREW_COMMAND="home";;
+  -S)          HOMEBREW_COMMAND="search";;
+  up)          HOMEBREW_COMMAND="update";;
+  ln)          HOMEBREW_COMMAND="link";;
+  instal)      HOMEBREW_COMMAND="install";; # gem does the same
+  rm)          HOMEBREW_COMMAND="uninstall";;
+  remove)      HOMEBREW_COMMAND="uninstall";;
+  configure)   HOMEBREW_COMMAND="diy";;
+  abv)         HOMEBREW_COMMAND="info";;
+  dr)          HOMEBREW_COMMAND="doctor";;
+  --repo)      HOMEBREW_COMMAND="--repository";;
```

Comment:
Also noticed too late: This is somewhat redundant given that `--repo` and `--repository` are handled further up. It is not completely redundant, because `brew --repo random garbage` still ends up calling the Ruby implementation in `cmd/--repository.rb` that we haven't eliminated yet. (Also stuff for a future PR, as we have to figure out how to reconcile this with `brew command` and `brew commands`.)